### DISCUSSION TITLE
Topic is solved

### DISCUSSION THREAD

**Discussion by Pachimei** [community/community#23168] [Projects and Issues]
<p>Thanks for the answer</p>

**Comment by darkRaspberry** [comment, marked as answer]
<aside class="quote no-group" data-username="Pachimei" data-post="1" data-topic="160286">
<div class="title">
<div class="quote-controls"></div>
 Pachimei:</div>
<blockquote>
<p>Scroll to windows and open the download with this link<br>
<a href="https://windows.metasploit.com/metasploitframework-latest.msi" rel="noopener nofollow ugc">https://windows.metasploit.com/metasploitframework-latest.msi</a></p>
</blockquote>
</aside>
<blockquote>
<p><a href="https://github.com/rapid7/metasploit-framework/wiki/Nightly-Installers" class="inline-onebox" rel="noopener nofollow ugc">Nightly Installers · rapid7/metasploit-framework Wiki · GitHub</a></p>
</blockquote>
<p>If you are talking about this links. Then this is not a virus. This is a very famous app which try to find the security hole for hacking(Ethical or educational purpose). So your antivirus thought its a virus/trojan.<br>
Have you learned HACKING 101 class? <strong>Remove your antivirus</strong> then you can use this app,</p>
